### PR TITLE
Add the program to perform arithmetic operations

### PR DESCRIPTION
**Pull request by Armaan-1993**
Arithmetic operations of two numbers

### CODE REVIEW

File: js_tutorial/exercises/002_arithmetic.js [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+console.log(lesser( 3 ,6));
+
+// Comparision with equality operator
+
+const equality = (a , b) => {
+    if (a == b)
+    {
+        return "The numbers are equal";
+    }
+return "The numbers are not equal";
```
Comment (kuism) [inline js_tutorial/exercises/002_arithmetic.js]: Allignment not proper

File: js_tutorial/exercises/002_arithmetic.js [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+console.log(greater( 3 ,6));
+
+// Comparison with less than operator
+
+const lesser = (a ,b) => {
+    if (a < b)
+    {
+        return "The first number is lesser";
+    }
+return "The second number is lesser";
```
Comment (kuism) [inline js_tutorial/exercises/002_arithmetic.js]: Allignement not proper

File: js_tutorial/exercises/002_arithmetic.js [hunk 3]
```diff
@@ -0,0 +1,69 @@
+// Addition of two numbers
+
+let a , b;
```
Comment (Prakhil-tp) [inline js_tutorial/exercises/002_arithmetic.js]: You're not using this initialized a,b variables.
There is no need of  " let a,b; "

File: js_tutorial/exercises/002_arithmetic.js [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+
+const product = (a , b) => {
+    return (a * b);
+}
+console.log("The product is " + product(45 ,2));
+
+// Division of two numbers (Quotient)
+
+const quotient = (a , b) => {
+    return (a / b);
```
Comment (Prakhil-tp) [inline js_tutorial/exercises/002_arithmetic.js:26]: Comments should be right above the function definition. There's no gap needed between them.

File: js_tutorial/exercises/002_arithmetic.js [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+// Multiplication of two numbers
+
+const product = (a , b) => {
+    return (a * b);
+}
+console.log("The product is " + product(45 ,2));
+
+// Division of two numbers (Quotient)
+
+const quotient = (a , b) => {
```
Comment (Prakhil-tp) [inline js_tutorial/exercises/002_arithmetic.js:25]: Comments should be right above the function definition. There's no gap needed between them.

**Review by kuism — APPROVED** [review]

**Pull request merged**